src/util/specific.py

```diff
@@ -60,6 +61,17 @@ def rename_custom_table(
     )
 
 
+def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
```

Comment:
```suggestion
def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
    """DOCSTRING"""
```

In principle, AFAI understood when we touch (or - like in this case - create) a util function, we should add/correct a DOCSTRING. So that, at some point, we will be able to automatically generate documentation. 

OTOH, I'm not sure if we have established a format yet, have we? @aj-fuentes @KangOl

Reply:
This function is simple enough.